config/locales/en/admin.yml

```diff
@@ -619,7 +619,6 @@ en:
           title: Proposals
           back: Back
           id: Id
-          title: Title
```

Comment:
I think in this case the proper thing would be to show "Proposals" as the title of the page and "Title" in the coumn header of the proposals table :thinking:.

What do you think?